experiments/AMIP/modular/components/ocean/eisenman_seaice.jl

```diff
@@ -0,0 +1,293 @@
+import ClimaCoupler: FluxCalculator
+import ClimaTimeSteppers as CTS
+import DiffEqBase: ODEProblem, init, step!
+import ClimaCoupler.FluxCalculator: update_turbulent_fluxes_point!, differentiate_turbulent_fluxes!
+import ClimaCoupler.Interfacer: get_field, update_field!
+
+# TODO: move to ClimaCore
+import Base.getindex
+Base.getindex(f::Fields.FieldVector, colidx) =
+    Fields.FieldVector(; zip(propertynames(f), map(x -> getproperty(f, x)[colidx], propertynames(f)))...)
+
+"""
+    EisenmanIceSimulation{P, Y, D, I}
+
+Thermodynamic 0-layer, based on the Semtner 1979 model and later refined by  sea ice model as specified by
+Eisenmen 2009 and Zhang et al 2021.
+"""
+struct EisenmanIceSimulation{P, Y, D, I} <: SeaIceModelSimulation
+    params_ice::P
+    Y_init::Y
+    domain::D
+    integrator::I
+end
+name(::EisenmanIceSimulation) = "EisenmanIceSimulation"
+
+Base.@kwdef struct EisenmanIceParameters{FT <: AbstractFloat}
+    h::FT = 2                       # sea ice height [m]
+    ρ::FT = 1500                    # sea ice density [kg / m3]
+    c::FT = 800                     # sea ice heat capacity [J / kg / K]
+    z0m::FT = 1e-3                  # roughness length for momentum [m]
+    z0b::FT = 1e-5                  # roughness length for buoyancy [m]
+    C0_base::FT = 120               # ice base transfer coefficient [W / m2 / K]
+    T_base::FT = 273.16             # ice base temperature [K]
+    L_ice::FT = 3e8                 # latent heat coefficient for ice [J / m3]
+    T_freeze::FT = 273.16           # temperature at freezing point [K]
+    k_ice::FT = 2                   # thermal conductivity of ice [W / m / K]
+    α::FT = 0.38                    # albedo
+    σ::FT = 5.67e-8                 # Stefan Boltzmann constant [W / m^2 / K^4]
+end
+
+Base.@kwdef struct EisenmanOceanParameters{FT <: AbstractFloat}
+    h::FT = 1                       # mixed layer depth [m]
+    ρ::FT = 800                    # density times heat transfer coefficient for mixed layer [kg / m^3]
```

Comment:
```suggestion
    ρ::FT = 800                     # density times heat transfer coefficient for mixed layer [kg / m^3]
```

Reply:
I didn't quite understand the description of this parameter... this variable represents ocean density times an O(1) parameter?

Reply:
Good catch! This should be just density and the value is wrong - now corrected (and value increased to 1020 kg/m3).